server/services/conditionsService.js

```diff
@@ -199,6 +199,31 @@ module.exports = function createConditionsService({ use2019Conditions }) {
     }
   }
 
+  function getNonStandardConditions(licence) {
+    const { licenceConditions } = populateLicenceWithConditions(licence)
+    const additionalConditionsContent = []
+    const pssConditionsContent = []
+    const bespokeConditionsContent = []
+
+    if (licenceConditions.length > 0) {
+      licenceConditions.forEach((condition) => {
+        if (condition.group === 'Bespoke' && condition.approved === 'Yes') {
+          bespokeConditionsContent.push(condition.content)
+        } else if (condition.group === 'Post-sentence supervision only') {
+          pssConditionsContent.push(condition.content)
+        } else if (condition.group !== 'Bespoke' || condition.group === undefined) {
+          additionalConditionsContent.push(condition.content)
+        }
+      })
```

Comment:
is the `licenceConditions.length` check required?

can you think how to simplify this and refactor it to use filter?

Reply:
the length check isn't needed
can use filter but the licenceConditions isn't always an array 
It is an object if no conditions are selected. This would then need to be converted to an array so .filter can be used. Something like this ...
   ``` if (typeof licenceConditions === 'object') {
      licenceConditions = Array.from(licenceConditions)
    }```